tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist =
+    py{27,33,34,35,36,37}
```

Comment:
not normally worth testing with all these pythons

i just test with 2.7 and 3.6 on my projects